docs/configuring-metabase/settings.md

```diff
@@ -8,54 +8,48 @@ redirect_from:
 
 This section contains settings for your whole instance, like its URL, the reporting timezone, and toggles for disabling or enabling some of Metabase's optional features.
 
-You can configure these settings in the **General** section of the **Settings** tab in the **Admin Panel**.
+You can configure these settings from **Settings** > **Admin Settings** > **General**.
 
 ## Site Name
 
 How you’d like to refer to this instance of Metabase.
 
 ## Site URL
 
-The base URL of this Metabase instance. The base URL is used in emails to allow users to click through to their specific instance. Make sure to include http:// or https:// to make sure it’s reachable.
+The site URL is the web address that people use to access your Metabase instance. Make sure to include `http://` or `https://` to make sure it’s reachable.
 
-## Redirect to HTTPS
+### Redirect to HTTPS
 
-Force all traffic to use HTTPS via redirect, if the site can serve over HTTPS.
+By default, Metabase is served over HTTP.
 
-_Default: disabled_.
+To force all traffic to use HTTPS via redirect, click `http://` (under the **Site URL** section) and select `https://` from the dropdown menu.
 
-For example, if you are serving your Metabase application at "example.com", and you enable HTTPS redirect, when a user enters an address like "example.com/data" in their browser's address bar, the user will be automatically redirected to a secure connection at `https://example.com/data`.
+For example, say you enable HTTPS redirect for a Metabase instance at the site URL "example.com". When someone enters an address like `example.com/data` in their browser's address bar, they'll get automatically redirected to a secure connection at `https://example.com/data`.
 
 > Note: if you haven't set up HTTPS on your server, Metabase will not let you enable HTTPS redirect. Instead, you'll get a warning saying "It looks like HTTPS is not properly configured."
 
 ## Email Address for Help Requests
 
 This email address will be displayed in various messages throughout Metabase when users encounter a scenario where they need assistance from an admin, such as a password reset request.
 
-## Approved domains for notifications
+## Approved Domains for Notifications
```

Comment:
I see you're trying to make this heading consistent with other headings, but our style guide prefers sentence case for headings. So these headings all need a haircut, even if the in-product casing suggests otherwise.